.travis.yml

```diff
@@ -29,6 +29,7 @@ env:
 install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
+  - pip install --upgrade python-magic
```

Comment:
Why not adding this to requirements.txt file?

Reply:
@tarteo, please, can you explain us this?